person/person.json

```diff
@@ -96,6 +96,53 @@
           }
         },
         {
+          "name": "activity",
+          "properties": {
+            "entity": {
+              "type": "string",
+              "enum": ["activity"],
+              "description": "Name of the entity to be queried."
+            },
+            "criteria": {
+              "type": "object",
+              "description": "Specific criteria for the activity entity.",
+              "properties": {
+                "network": {
+                  "type": "string",
+                  "description": "Name of a network."
+                },
+                "date": {
+                  "type": "object",
+                  "description": "The date of an activity entity.",
+                  "properties": {
+                    "from": {
+                      "type": "string",
+                      "description": "From date of date range for the activity entity. ISO date string."
+                    },
+                    "to": {
+                      "type": "string",
+                      "description": "To date of date range for the activity entity. ISO date string."
+                    }
+                  }
+                },
+                "type": {
+                  "type": "string",
```

Comment:
Should we have an `"enum": ["used","updated"],` here? I am assuming the default of 'either' you mention is when the `type` property is not present?